### DISCUSSION TITLE
Cool Project

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [jeffreyvr/dropblockeditor#21] [Q&A]
Hey @jeffreyvr, pretty cool project. something that I have been looking for quite sometime. And it using Livewire is a sweet spot. 
Got a couple of questions;
- Any chance of a discord channel for questions, suggestions and feedback?
- Is there a way to group blocks?

**Comment by jeffreyvr** [comment]
Thanks @abishekrsrikaanth!

For the time being I think that the discussion board here will do, but certainly an option to open up a discord in the future.

At the moment grouping is not possible. Though, I'm doing some experimenting with groups/nested blocks which is looking promising so far.

**Reply by abishekrsrikaanth** [reply]
Sounds good. Glad to hear about grouping. I know you have all the ideas for the project, but I would love to be able to contribute as well when the project is a bit more stable and feature-ready. I love it though.

**Comment by abishekrsrikaanth** [comment]
One of the features that I would love to have is the option to bind dynamic model data to blocks. Not quite sure how to do that. Let's say binding FAQ data from the database to a block.

**Reply by jeffreyvr** [reply]
You can set the initial data in the block.

```php
class Faqs extends Block
{
    public $title = 'Article';

    public function __construct()
    {
        $this->data['items'] = App\Models\Faq::all()->toArray();
    }
```

Then you can use `$items` inside the render.

**Reply by abishekrsrikaanth** [reply]
So, if I run this through the parser, it will output the html with the data from the time we save the data, how do we go about rendering the html always with the latest data? Does it make sense?

**Reply by jeffreyvr** [reply]
Ah I see, makes sense. Will have to think about that some more.
